aws/scenarios/vm/ec2VM/ec2VM.go

```diff
@@ -3,17 +3,12 @@ package ec2vm
 import (
```

Comment:
Indeed, this is something I already listed in my improvement list. Tracked by AETT-66 to avoid conflicts with opened PRs.
Note: I didn't changed the path of the file in this PR.